src/app/common/KFilterBar.vue

```diff
@@ -149,7 +149,8 @@ const props = defineProps({
    */
   id: {
     type: String,
-    required: true,
+    required: false,
+    default: 'data-plane-proxy-filter',
```

Comment:
**Suggestion**: This shouldn’t be here. This component is completely agnostic to its usage context and should remain so. It’s no problem setting this on use.